src/test/java/org/threeten/extra/chrono/TestInternationalFixedChronology.java

```diff
@@ -0,0 +1,1191 @@
+                // year day / leap day are not part of any months!
+                continue;
+            }
+
+            y = (i > 169 && date.isLeapYear()) ? i - 1 : i;
+            dayOfMonth = ((y - 1) % 28) + 1;
+            month = ((y - 1) / 28) + 1;
+
+            assertEquals(month, date.getMonth());
+            assertEquals(dayOfMonth, date.getDayOfMonth());
+        }
+    }
+
+    //-----------------------------------------------------------------------
+    // InternationalFixedDate.isLeapYear
+    //-----------------------------------------------------------------------
+    @Test
+    public void test_isLeapYear_loop() {
+        Predicate<Integer> isLeapYear = year -> {
+            return ((year & 3) == 0) &&((year % 100) != 0 ||(year % 400) == 0);
+        };
+
+        for(int year = 1; year < 1001; year++) {
+            InternationalFixedDate base = InternationalFixedDate.of(year, 1, 1);
+            assertEquals(base.isLeapYear(), isLeapYear.test(year), "Year " + year + " is failing");
+            assertEquals(InternationalFixedChronology.INSTANCE.isLeapYear(year), isLeapYear.test(year), "Year " + year + " is failing leap-year test");
+        }
+    }
+
+    @Test
+    public void test_isLeapYear_specific() {
+        assertEquals(InternationalFixedChronology.INSTANCE.isLeapYear(400), true);
+        assertEquals(InternationalFixedChronology.INSTANCE.isLeapYear(100), false);
+        assertEquals(InternationalFixedChronology.INSTANCE.isLeapYear(99), false);
+        assertEquals(InternationalFixedChronology.INSTANCE.isLeapYear(7), false);
+        assertEquals(InternationalFixedChronology.INSTANCE.isLeapYear(6), false);
+        assertEquals(InternationalFixedChronology.INSTANCE.isLeapYear(5), false);
+        assertEquals(InternationalFixedChronology.INSTANCE.isLeapYear(4), true);
+        assertEquals(InternationalFixedChronology.INSTANCE.isLeapYear(3), false);
+        assertEquals(InternationalFixedChronology.INSTANCE.isLeapYear(2), false);
+        assertEquals(InternationalFixedChronology.INSTANCE.isLeapYear(1), false);
+    }
+
+    //-----------------------------------------------------------------------
+    // InternationalFixedDate.era
+    // InternationalFixedDate.dateYearDay
+    // InternationalFixedDate.prolepticYear
+    //-----------------------------------------------------------------------
+    @DataProvider(name = "lengthOfMonth")
+    Object[][] data_lengthOfMonth() {
+        return new Object[][] {
+                { 1900,  1, 28 },
+                { 1900,  2, 28 },
+                { 1900,  3, 28 },
+                { 1900,  4, 28 },
+                { 1900,  5, 28 },
+                { 1900,  6, 28 },
+                { 1900,  7, 28 },
+                { 1900,  8, 28 },
+                { 1900,  9, 28 },
+                { 1900, 10, 28 },
+                { 1900, 11, 28 },
+                { 1900, 12, 28 },
+                { 1900, 13, 28 },
+
+                { 1901, 13, 28 },
```

Comment:
This is the set I was referring to.  Month 13 in this calendar will _always_ be 28 days, so testing like this is pointless.  You need to be testing the "length" of the leap-day and year-day months.  You may need to choose a different year, though, given 1900 isn't a leap-year here.